src/Native/pkg/runtime.native.System.IO.Compression/runtime.native.System.IO.Compression.pkgproj

```diff
@@ -43,15 +55,6 @@
     <ProjectReference Include="win8\runtime.native.System.IO.Compression.pkgproj">
       <Platform>arm</Platform>
     </ProjectReference>
-    <ProjectReference Include="win10\runtime.native.System.IO.Compression.pkgproj">
```

Comment:
or does it only reference the endpoints i.e. the pkgproj files that specify the NativePath?